src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/BundleSlingInitialContentJarEntryExtractor.java

```diff
@@ -117,17 +115,21 @@ void extractAndAddToAssembler(@NotNull BundleSlingInitialContentExtractContext c
 
             }
 
-            try (Archive virtualArchive = SingleFileArchive.fromPathOrInputStream(tmpDocViewInputFile, bundleFileInputStream,
-                    () -> Files.createTempFile(context.getConverter().getTempDirectory().toPath(), "initial-content", Text.getName(file.getName())), contentPackageEntryPath)) {
-                // in which content package should this end up?
-
-                if (tmpDocViewInputFile != null) {
-                    packageAssembler.addEntry(contentPackageEntryPath, tmpDocViewInputFile.toFile());
-                } else {
-                    packageAssembler.addEntry(contentPackageEntryPath, bundleFileInputStream);
+            //if we are writing in a .content.xml file as default, 
+            // but the .content.xml is already present (written by specifications, then we don't do anything.)
```

Comment:
i think the ) needs to go in a different place to make it clear, no?

`// but the .content.xml is already present (written by specifications), then we don't do anything.`